user-org-util/pom.xml

```diff
@@ -51,5 +51,16 @@
             <artifactId>akka-slf4j_2.11</artifactId>
             <version>2.5.16</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.typesafe.play</groupId>
```

Comment:
why util will have com.typesafe.play library?

Reply:
sir because of Mapper class which is present in util .